client/components/Item/UpNext.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <v-fade-transition>
+    <v-card class="container white--text pa-4">
+      <div class="d-flex flex-column flex-grow-1">
+        <v-card-title class="text-h6 pa-0 my-1 mx-0">
+          {{ $t('dialog.upNext.nextItemPlayingIn') }}
+          <span class="primary--text darken-2">
+            &ensp;{{ timeLeft }} {{ $t('seconds').toLowerCase() }}
+          </span>
+        </v-card-title>
+        <v-card-subtitle
+          v-if="getCurrentItem.Type === 'Episode'"
+          class="mt-1 mx-0 mb-2 text-truncate subtitle-1 pa-0"
+        >
+          {{ getNextItem.SeriesName }} -
+          {{
+            $t('seasonEpisodeAbbrev', {
+              seasonNumber: getNextItem.ParentIndexNumber,
+              episodeNumber: getNextItem.IndexNumber
+            })
+          }}
+          <span v-if="$vuetify.breakpoint.smAndUp"> - </span> <br v-else />
+          {{ getNextItem.Name }}
+        </v-card-subtitle>
+        <v-card-subtitle
+          v-if="getCurrentItem.Type === 'Movie'"
+          class="mt-1 mx-0 mb-2 text-truncate subtitle-1 pa-0"
+        >
+          {{ getNextItem.Name }}
+        </v-card-subtitle>
+        <div>
+          {{ getRuntimeTime(getNextItem.RunTimeTicks) }}
+          <span class="pl-4"
+            >{{ $t('endsAt', { time: nextEndsAt(getNextItem.RunTimeTicks) }) }}
+          </span>
+        </div>
+        <v-card-actions>
+          <v-spacer />
+          <v-btn class="primary darken-2" @click="$emit('startNext')">
```

Comment:
Prefer kebab-case for event names (It's the recommended convention for Vue 2).

But also, why send an event upwards? You could simply map the action too and call it directly.